nginx.master.conf

```diff
@@ -158,21 +158,7 @@ http {
         }
 
         location ~ ^/(slave|agent)/(?<agentid>[0-9a-zA-Z-]+)(?<url>.*)$ {
-            access_by_lua 'auth.validate_jwt_or_exit()';
-            set $agentaddr '';
-            set $agentport '';
-
-            more_clear_input_headers Accept-Encoding;
-            rewrite ^/(slave|agent)/[0-9a-zA-Z-]+/.*$ $url break;
-            rewrite_by_lua_file conf/master/agent.lua;
-
-            include common/proxy-headers.conf;
-            include common/server-sent-events.conf;
-
-            # Supports chunked encoding and stream for debugging API
-            proxy_request_buffering off;
-
-            proxy_pass http://$agentaddr:$agentport;
+            rewrite ^(slave|agent)/$agentid/$url $scheme://system/v1/agent/$agentid/mesos/$url permanent;
```

Comment:
Can you please ask @spahl and @jgehrcke for a blessing for that? The `/service` endpoint was with us for a very long time already, and I am not sure if we want to redirect everything like this.